rero/rero-ils-ui
# request: fix request made by a librarian

* Changes messages.
* Improves button appearance.
* Disables 'delete' button when a request is done.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>

## Why are you opening this PR?

To implement task 1353 of US657
https://tree.taiga.io/project/rero21-reroils/task/1353?kanban-status=1224894
See https://gitter.im/rero/reroils-PO?at=5e65f10102dbe022a175fa76

## How to test?

1. Go to the detailed view of a document of your library.
0. Buttons' design should be uniformized with other buttons of the app.
0. Messages should match PO specifications (see https://gitter.im/rero/reroils-PO?at=5e65f10102dbe022a175fa76 or https://tree.taiga.io/project/rero21-reroils/task/1353?kanban-status=1224894)
2. Make a request for a patron.
3. When it's done, the delete button should be dark and delete should be forbidden.


## Code review check list
